users/views_api.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import generics, views, permissions
+from . import serializers
+
+
+class UsersView(generics.CreateAPIView):
+    serializer_class = serializers.UserSerializer
+    permission_classes = (permissions.AllowAny,)
+
+
+class UserDetailView(generics.RetrieveUpdateAPIView):
+    serializer_class = serializers.UserDetailSerializer
```

Comment:
According to your default permission settings. Anonymous users (not authenticated) can visit this endpoint.